src/components/SRNEntry.vue

```diff
@@ -143,20 +124,19 @@ export default {
       const userID = parseInt(this.userIDList["0"]["userID"]);
 
       //response tells us if the user is authenticated.
-      this.validated = await firebaseAPI.checkUserExists(userID);
+      const userIsValidated = validateUser(
```

Comment:
It's not a good practice to use `const` for objects.
Here, `userIsValidated` is an object (yes, arrays are objects too).
`const` variables cannot be unassigned but they can be mutated for example `userIsValidated[2] = "new value"`
This statement is valid. So there's no point to use `const`
You should use simply `let`